Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof

```diff
@@ -0,0 +1,59 @@
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphRegistrationEnforcement
+{
+    [Write, Description("Run campaigns to remind users to setup targeted authentication methods."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodsRegistrationCampaign")] String AuthenticationMethodsRegistrationCampaign;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphAuthenticationMethodsRegistrationCampaign
+{
+    [Write, Description("Users and groups of users that are excluded from being prompted to set up the authentication method."), EmbeddedInstance("MSFT_MicrosoftGraphExcludeTarget")] String ExcludeTargets[];
+    [Write, Description("Users and groups of users that are prompted to set up the authentication method."), EmbeddedInstance("MSFT_MicrosoftGraphAuthenticationMethodsRegistrationCampaignIncludeTarget")] String IncludeTargets[];
+    [Write, Description("Specifies the number of days that the user sees a prompt again if they select 'Not now' and snoozes the prompt. Minimum 0 days. Maximum: 14 days. If the value is '0'  The user is prompted during every MFA attempt.")] UInt32 SnoozeDurationInDays;
+    [Write, Description("Enable or disable the feature. Possible values are: default, enabled, disabled, unknownFutureValue. The default value is used when the configuration hasn't been explicitly set and uses the default behavior of Azure AD for the setting. The default value is disabled."), ValueMap{"default","enabled","disabled","unknownFutureValue"}, Values{"default","enabled","disabled","unknownFutureValue"}] String State;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget
```

Comment:
To avoid conflicts across resources in the future, we took the decision to always rename these CIMInstances as:

MSFT_resourceNameCIMInstance

Please review every schema.mof and module files in this Pull Request to adhere to this naming standard. Thanks